deploy/kubernetes/overlays/windows/dev/drivertest.yaml

```diff
@@ -0,0 +1,72 @@
+kind: DaemonSet
```

Comment:
is this file used?

Reply:
no, I think I added the whole dir and forgot to remove those tmp files. Removed them.